### PR TITLE
release: v1.0.0-alpha.59

### PR DESCRIPTION
**Pull request by Pineirin**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Changes
 
+Version 1.0.0-alpha.59 (released 2021-12-02)
+
+- Validate that vocabulary fields passed are valid.
```
Comment (kprzerwa) [inline CHANGES.md:5]:
```suggestion
- Validate that vocabulary fields values in the form select
```

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+- Ensure that the first loan of each document is the correct one.
```
Comment (kprzerwa) [inline CHANGES.md:6]: not sure what is the correct loan ?

File: CHANGES.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+- Properly order all loans.
```
Comment (kprzerwa) [inline CHANGES.md:7]:
```suggestion
- Order loans in search by ascending creation date
```

File: CHANGES.md [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+- Fix 'See all' button in the loans history.
```
Comment (kprzerwa) [inline CHANGES.md:8]:
```suggestion
- Fix 'See all' button search query in the loans history.
```

File: CHANGES.md [hunk 5]
```diff
@@ -1,5 +1,15 @@
...
+- Allow downloading loans in csv/json.
```
Comment (kprzerwa) [inline CHANGES.md:9]:
```suggestion
- Allow downloading loans in csv/json file.
```

File: CHANGES.md [hunk 6]
```diff
@@ -1,5 +1,15 @@
...
+- Warn the users that try to extend an overbooked loan.
```
Comment (kprzerwa) [inline CHANGES.md:10]:
```suggestion
- Add warning for the librarian tring to extend an overbooked loan.
```

File: CHANGES.md [hunk 7]
```diff
@@ -1,5 +1,15 @@
...
+- Fix error handling.
```
Comment (kprzerwa) [inline CHANGES.md:11]:
```suggestion
- Add error notification for unhandled API errors.
```

**Review by zzacharo — APPROVED** [review]

**Review by ntarocco — APPROVED** [review]

**Pull request merged**